helm/kciss/values.yaml

```diff
@@ -5,6 +5,7 @@
 kciss:
   replicaCount: 1
   port: 9300
+  targetPort: 9300
```

Comment:
need a change in deployment as well (L24)

Reply:
I did not use the targetPort on the deployment on purpose so we can add the side car container and route all the traffic from the ingress to the sidecar using the targetPort.

Reply:
ok, so the Helm should support both cases:
- Ext -> Ingress -> Service -> Pod Container Side-Car Port (kciss.targetPort) -> Pod Container Kciss Port (kciss.port)
- Prom (int) -> Service -> Pod Container Kciss Port (kciss.port)
From what I see, the proposal would break the case #2 (service using kciss.targetPort and deployment using kciss.port), or did I miss anything ?

Reply:
ok, so the Helm should support both cases:
- Ext -> Ingress -> Service -> Pod Container Side-Car Port (kciss.targetPort) -> Pod Container Kciss Port (kciss.port)
- Prom (int) -> Service -> Pod Container Kciss Port (kciss.port)
From what I see, the proposal would break the case #2 (service using kciss.targetPort and deployment using kciss.port), or did I miss anything ?

Reply:
Both cases are covered:
1) ingress (80) --> service(9300) --> SideCar (targetPort 9301) --> KCISS pod (9300)
2) ingress (80) --> service(9300) --> KCISS pod (targetPort 9300)

The targetPort is map between the service exposed port (9300) and the pod's exposed port (by default it's kciss on 9300, but could be the side car on port 9301)

Reply:
ok, so I think the misunderstanding comes from the fact that there'll be 2 ports (kciss.port and kciss.targetPort) and using 2 different values would break the simple case. But I don't see an easy way to make both cases coexist. I'll accept as is and document that.